cosmoz-page-router.js

```diff
@@ -105,16 +105,15 @@
 		* @return {Boolean} `true` = continue, `false` = prevent further actions
 		*/
 		_fireEvent(type, detail, node, bubbles) {
-			return !this.dispatchEvent(new CustomEvent(
+			return this.dispatchEvent(new CustomEvent(
 				type,
 				{
 					bubbles: !!bubbles,
 					cancelable: true,
 					composed: true,
-					detail,
-					node: node || this
```

Comment:
Why aren't we interested in node?

Reply:
@nomego I'm not sure it works anymore with CustomEvent. Do you know where it is used?